study-builder/studies/rgp/i18n/en.conf

```diff
@@ -1,11 +1,12 @@
 {
   //Prequal
   "prequal_name": "Tell us your story",
-  "prompt_prequal_question": "I am, or am applying on behalf of, a patient who: ",
-  "prequal_understands_english": "Understands English",
-  "prequal_lives_us": "Lives in the United States",
-  "prequal_has_condition": "Has a rare and genetically undiagnosed condition",
-  "prequal_has_doctor": """Is currently under the care of a doctor that is helping to
+  "prompt_general_info": "I am, or am applying on behalf of, a patient who: ",
+  "general_info_english": "Understands English",   // or Spanish?
```

Comment:
You're right: this should be "Understands English or Spanish"

Reply:
Fixed!